src/parser/ast/declares.rs

```diff
@@ -0,0 +1,57 @@
+use serde::Deserialize;
+use serde::Serialize;
+
+use crate::lexer::token::Span;
+use crate::parser::ast::identifiers::SimpleIdentifier;
+use crate::parser::ast::Expression;
+use crate::parser::ast::Statement;
+
+#[derive(Debug, PartialEq, Eq, Clone, Deserialize, Serialize)]
+#[serde(rename_all = "snake_case")]
+pub struct DeclareEntry {
+    pub key: SimpleIdentifier,
+    pub span: Span,
+    pub value: Expression,
+}
+
+#[derive(Debug, PartialEq, Eq, Clone, Deserialize, Serialize)]
+#[serde(rename_all = "snake_case")]
+pub struct DeclareEntryGroup {
+    pub start: Span,
+    pub end: Span,
+    pub entries: Vec<DeclareEntry>,
+}
```

Comment:
`start` -> `(`
`end` -> `)`
`entries` -> list of `k=v`.

we still don't keep track of comma positions, but that is planned for later.